build.xml

```diff
@@ -478,6 +481,35 @@
         </exec>
     </target>
 
+    <!-- Include annotation processor for VirtualTable -->
```

Comment:
encapsulate this annotation processor to `.build/build-compile-annotations.xml` ?

Reply:
Done!